# Example message file
string maker_name             # placeholder for manufacturer name
uint32 part_number            # placeholder for part number of the sensor
